python/tests/app_interscalehub.py

```diff
@@ -21,7 +21,7 @@ def run_wrapper(args):
     # 1 --> nest to Tvb
     # 2 --> tvb to nest
     param = Parameter()
-    direction = int(args[1])
+    direction = int(args)
```

Comment:
I would put a try catch around this. int() fails hard

Reply:
I add a NOTE. For the moment this is a temporary solution and will be changed in the next iteration (proper interface for steering).